data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg

```diff
@@ -16,10 +16,10 @@
     [story]
         [part]
             music=loyalists.ogg
-            story= _ "Laden with gold and filled with an elation at their victory that was tempered with sadness at the death of their friends, Tallin and his people made their way back to the mines."
+            story= _ "Laden with gold, and filled with not only elation at their victory, but sadness for their departed comrades, Tallin and his people made their way back to the mines."
         [/part]
         [part]
-            story= _ "As they approached they were met with a most welcome sight: new fortifications, looking very much like the legendary impenetrable dwarvish defenses of old, had sprung up everywhere. All of them were manned by alert and well-armed dwarves who hailed them boisterously. The mines were belching smoke from the furiously working forges. New weapons and armor for humans and dwarves alike lay neatly stacked, and the deafening clang of hammers and picks from the dwarves promised more."
+            story= _ "As they approached they were met with a most welcome sight: new fortifications, looking very much like the legendary, impenetrable dwarvish defenses of old, had sprung up everywhere. All of them were manned by alert and well-armed dwarves who hailed them boisterously. Smoke belched from furiously working forges and the heat from the smelters was nearly staggering. New weapons and armor for humans and dwarves alike lay neatly stacked, and the deafening clang of hammers and picks from the dwarves promised more."
```

Comment:
The first comma is unnecessary I think but the rest is great.

Reply:
Is it legendary and impenetrable, or is it t_the_ legendary "impenetrable dwarvish defenses" TM?

Reply:
Since this isn't a formal document with stricter grammar rules, I air on the side of readability and speaking patterns especially if it's dialogue. So just because it's grammatically correct doesn't mean it's a change we should make.

Just FYI for further text revisions.

Reply:
I understand that. This isn't my first time editing stories.

However, A comma shouldn't affect readability.

Reply:
For sure, but I do think commas impact readability since people naturally tend to pause when encountering them. I don't think it's a huge deal either way but depends on the author's style. So if it's between the author and an editor I will prefer to go with the author's version.

Reply:
Either way, it isn't a big step-down in editing. I changed it.